huaweicloud/resource_huaweicloud_networking_secgroup_v2.go

```diff
@@ -98,28 +116,40 @@ func resourceNetworkingSecGroupV2Create(d *schema.ResourceData, meta interface{}
 
 	d.SetId(security_group.ID)
 
+	description := d.Get("description").(string)
+	networkingClient3, err := config.NetworkingV2Client(GetRegion(d, config))
```

Comment:
maybe we can have more meaningful names rather than networkingClient2 and networkingClient3

Reply:
fixed